time stamp
float32[] command_before_limiting
float32[] command 
float32[] read_position
float32[] read_velocity 
float32[] read_effort
float32[] read_current


